src/CitMovie.Data/FollowRepository/FollowRepository.cs

```diff
@@ -0,0 +1,60 @@
+using CitMovie.Models.DomainObjects;
+using Microsoft.EntityFrameworkCore;
+
+namespace CitMovie.Data.FollowRepository
+{
+    public class FollowRepository : IFollowRepository
+    {
+        private readonly FrameworkContext _context;
+
+        public FollowRepository(FrameworkContext context)
+        {
+            _context = context;
+        }
+
+        public async Task<IList<Follow>> GetFollowings(int userId, int page, int pageSize)
+        {
+            return await _context.Follows
+                .Where(f => f.UserId == userId)
+                .Skip(page * pageSize)
+                .Take(pageSize)
+                .ToListAsync();
+        }
+
+        public async Task<int> GetTotalFollowingsCount(int userId)
+        {
+            return await _context.Follows.CountAsync(f => f.UserId == userId);
+        }
+
+        public async Task<Follow> CreateFollow(int userId, int personId)
+        {
+            var follow = new Follow
+            {
+                UserId = userId,
+                PersonId = personId,
+                FollowedSince = DateTime.UtcNow
+            };
```

Comment:
I would think that this should be part of the business layer since and then `CreateFollow` should just take a `User` argument which it uses to create the row.

Reply:
Yes. I will try implementing this so that it is in the business layer, and then the dto is just returned from there.